generics/kafka/producers.js

```diff
@@ -3,83 +3,108 @@
  * author : Aman Karki
  * created-date : 08-Sep-2020
  * Description : Kafka Producer related information.
-*/
+ */
 
 // Dependencies
-const kafkaCommunicationsOnOff = (!process.env.KAFKA_COMMUNICATIONS_ON_OFF || process.env.KAFKA_COMMUNICATIONS_ON_OFF != "OFF") ? "ON" : "OFF";
-const projectSubmissionTopic = (process.env.PROJECT_SUBMISSION_TOPIC && process.env.PROJECT_SUBMISSION_TOPIC != "OFF") ? process.env.PROJECT_SUBMISSION_TOPIC : "sl-improvement-project-submission-dev";
+const kafkaCommunicationsOnOff =
+	!process.env.KAFKA_COMMUNICATIONS_ON_OFF || process.env.KAFKA_COMMUNICATIONS_ON_OFF != 'OFF' ? 'ON' : 'OFF'
+const projectSubmissionTopic =
+	process.env.PROJECT_SUBMISSION_TOPIC && process.env.PROJECT_SUBMISSION_TOPIC != 'OFF'
+		? process.env.PROJECT_SUBMISSION_TOPIC
+		: 'sl-improvement-project-submission-dev'
+const userProjectActivityTopic =
+	process.env.USER_ACTIVITY_TOPIC && process.env.USER_ACTIVITY_TOPIC != 'OFF'
+		? process.env.USER_ACTIVITY_TOPIC
+		: 'dev-user-activities'
```

Comment:
@praveenKDass you asked to use here. ok please make it user-activities

Reply:
Added @VISHNUDAS-tunerlabs